src/routes/Auth.ts

```diff
@@ -0,0 +1,26 @@
+import express, { Router, Request, Response } from 'express';
+import config from '../config/config';
+
+import DAuthLogin from '../controllers/Auth/DAuthLogin';
+
+const router: Router = express.Router();
+
+router.get('/', async (req: Request, res: Response) => {
+  res.redirect('/student/login/dauth');
+});
+
+router.get('/student/login/dauth', (req, res) => {
+  res.redirect(
+    `https://auth.delta.nitt.edu/authorize?client_id=${
+      config.dauth.DAUTH_CLIENT_ID
+    }&redirect_uri=${encodeURIComponent(
+      config.dauth.DAUTH_CALLBACK_URL
+    )}&response_type=code&grant_type=authorization_code&state=sdafsdghb&scope=email+openid+profile+user&nonce=${
+      config.dauth.DAUTH_NONCE
+    }`
+  );
+});
+
+router.get('/dauth/callback', DAuthLogin);
```

Comment:
![image](https://user-images.githubusercontent.com/63252119/147734445-2219c2f6-5f89-44cc-84af-6a77a8c02c11.png)

Upon declining access, handle the error using a toast container or separate access denied component.